_posts/2017-06-06-inklink.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "InkLink"
+date:   2017-06-06 09:30:30 +0100
+categories: jekyll
```

Comment:
The categories are arbitrary. This is a hangover from the file that I copied to base this post on. The only place it appears at current is on the home page card. I'll change it to "experiment".